static/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -26,6 +26,20 @@
                         <i class="fa fa-remove" aria-hidden="true"></i>
                     </button>
                 </div>
+                <div class="nothing-to-show">
+                    <div class="view_all_stream_link_container">
+                        <span>
+                            {{t 'You are not subscribed to any streams.'}} <a href="#streams/all">{{t 'View all streams'}}</a>
+                        </span>
+                    </div>
+                    <div class="create_stream_link_container">
+                        {{#if can_create_streams}}
+                        <span>
+                            {{t 'There are no streams you can view in this organization.'}} <a class="create_new_stream_link" href="#streams/new">{{t 'Create a stream'}}</a>
```

Comment:
We need to show this text "There are no streams ...." even when user cannot create streams, we just need to hide the link "Create a stream" in case user is not allowed to create new streams.